### PR TITLE
Project Deliverable 2

### PR DESCRIPTION
**Pull request by specialcanadian**
Minor additions in main script. 

Added Google slides link in the readme

### CODE REVIEW

File: MusicClassification.ipynb [hunk 1]
```diff
@@ -1068,25 +1078,46 @@
...
       ]
      },
-     "execution_count": 26,
+     "execution_count": 27,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "tree_model.classes_"
```
Comment (jsheppard95) [inline MusicClassification.ipynb:1096]: Let's take this cell out. It's not currently doing anything/looks like was used for debug purposes.

**Review by jsheppard95 — COMMENTED** [review]
This looks great! Thanks for adding the CSV data loading lines to the `Load_Data` notebook. Nice to have that all in one place. My only comment was about an unnecessary cell that looks like was used for debug purposes. If you just delete that cell, make one more commit to your branch with the change, then it will automatically show up here and I'll merge the PR.

**Comment by jsheppard95** [comment]
Also I just reran the notebooks from my computer and got normal results, i.e not 99% accuracy for every model.

**Review by Kyle2Miles93 — COMMENTED** [review]
This looks good! I appreciate the google slide link, and it seems like a thorough enough scaffold of our project premise.

**Pull request merged**